### PR TITLE
Make App ID and App expiration dates more specific

### PR DESCRIPTION
**Pull request by junepark678**
### Changes

- Use `DateComponentsFormatter` to format expiration dates

### Todo before merge

- [x] Write changes
- [x] Test

### CODE REVIEW

**Comment by junepark678** [comment]
image
image
image
playground im using:
```swift
import Foundation
var currentDate = Date()

let formatter = DateComponentsFormatter()
formatter.unitsStyle = .full
formatter.includesApproximationPhrase = false
formatter.includesTimeRemainingPhrase = false

formatter.allowedUnits = [.day, .hour, .minute]

formatter.maximumUnitCount = 1

print(formatter.string(from: currentDate, to: Date(timeIntervalSinceNow: 3600 * 24))!)
```

**Review by SternXD — APPROVED** [review]
So I have looked at this and I approve so far but I'll wait for another member of the SideStore org to approve too before I merge.

**Comment by junepark678** [comment]
bruh, accidentally pressed that button

**Pull request merged**
